src/SignalR/clients/ts/signalr/src/HubConnection.ts

```diff
@@ -854,7 +854,7 @@ export class HubConnection {
 
     private isObservable(arg: any): arg is IStreamResult<any> {
         // This allows other stream implementations to just work (like rxjs)
-        return arg.subscribe && typeof arg.subscribe === "function";
+        return arg && arg.subscribe && typeof arg.subscribe === "function";
```

Comment:
Null refs

Reply:
Good catch. Is there a regression test?

Reply:
I haven't written any tests as yet. I'll get a few done once I had the design out of the way